### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by maximekabel**
Home page has been completely  redesigned

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: cosmos/templates/footer.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                <div class="col-md-4 d-none text-center d-md-block">
+                    <img src="{% static 'cosmos/img/cosmos_alternate_grey.svg' %}">
+                </div>
+                <div class="col-md-4 text-center py-2">
+                    <h4>Useful links</h4>
+                    <p>
+                        <a href="#">Privacy Policy</a> <br>
+                        <a href="#">Terms and Conditions</a> <br>
+                        <a href="#">Eindhoven Student Wiki</a> <br>
+                        <a href="#">Linktree</a> <br>
```
Comment (dasyad00) [inline cosmos/templates/footer.html:30]: add missing links pls

File: cosmos/templates/index.html [hunk 2]
```diff
@@ -1 +1,156 @@
...
+            <img class="img-fluid mh-50" src="{% static 'cosmos/img/cosmos_white_horizontal.png' %}">
+        </div>
+    </div>
+</section>
+<section id="stats" class="section-bg">
+    <div class="container py-5">
+        <div class="row">
+            <div class="col-lg-3 col-6">
+                <div class="stat-box">
+                    <span>69</span>
```
Comment (dasyad00) [inline cosmos/templates/index.html:14]: should we add the functions to count the number of nationalities in the same PR or separate PR (likewise for the no of members, active years, #events per year) @bcvandendool @maximekabel

**Review by dasyad00 — CHANGES_REQUESTED** [review]

File: cosmos/templates/index.html (hunk 2 above)
Comment (bcvandendool) [reply cosmos/templates/index.html:14]: I think we can do that in a separate pr, cause rn the branches are getting too out of sync, so i want these pull requests to get merged asap

Comment (maximekabel) [reply cosmos/templates/index.html:14]: I think we should add it later, I don't think it's useful to spend time on smth like this rn

File: cosmos/templates/footer.html (hunk 1 above)
Comment (maximekabel) [reply cosmos/templates/footer.html:30]: We'll do this in another branch when we have a privacy policy page and T&C page

**Review by dasyad00 — APPROVED** [review]

**Pull request merged**